torchio/data/image.py

```diff
@@ -117,11 +118,25 @@ def __init__(
             tensor: Optional[TypeData] = None,
             affine: Optional[TypeData] = None,
             check_nans: bool = False,  # removed by ITK by default
-            reader: Callable = read_image,
+            reader: Callable = None,
             **kwargs: Dict[str, Any],
             ):
         self.check_nans = check_nans
-        self.reader = reader
+
+        if reader is not None:
+            self.reader = reader
+        elif 'dims_bioformat' in kwargs:
```

Comment:
I might consider turning this block into a function itself that is static on the object.

```python
img = Image(..., reader=Image.aicsimageio_reader)
```

That way you aren't changing the defaults at all, you would just need to update the reader code to store the requested dims somewhere?